### DISCUSSION TITLE
A few clarifications

### DISCUSSION THREAD

**Discussion by sergiolamares** [kubilus1/autoortho#118] [Q&A]
I'm not understanding how autoortho works, and I have a few questions:
1. What are the inicial continental packages for? are they mandatory?
2. If so, when does autoortho downloads tiles on the fly acording to one's flight?
3. On the CMD window i'm always getting a message saying "INFO:flighttrack:We are connected but a packet timed out.  NBD." what does it mean?
4. whats the "z_autoortho" folder for?
5. Is python needed?
6. Can't find requirements.txt file as mentioned in the quick start guide
7. How do I know it's working properly?
Thanks.

**Comment by evilstiefel** [comment]
1. What are the inicial continental packages for? are they mandatory?
The terrain in X-Plane has two parts: a 3D-mesh defining elevation and then there are the textures that are getting "draped over" that mesh so you can see the ground. Otherwise, it would basically just be a wireframe. The downloads for the continents contain that mesh and are required for the way autoortho later on downloads the textures to be laid over that mesh information.

2. If so, when does autoortho downloads tiles on the fly acording to one's flight?
It does that when you're loading into the sim. That's the reason you need to click on "Run" and have the little autoortho-window run in the background while flying

3. On the CMD window i'm always getting a message saying "INFO:flighttrack:We are connected but a packet timed out. NBD." what does it mean?
It basically means that the flight tracker from autoortho could not get the current airplane position from the sim. NBD means "no big deal", you can safely ignore it. The message will escalate if autoortho can't communicate with the sim for a longer time.

4. whats the "z_autoortho" folder for?
That folder contains a "textures" folder which appears to be empty at first glance. That is the mounting point for the filesystem that autoortho creates when it's running. Don't delete it and don't worry about it, it's internal plumbing.

5. Is python needed?
No, if you are running the executable file on Windows, you don't need to have Python installed

6. Can't find requirements.txt file as mentioned in the quick start guide
That file is part of the git repository. If you are not a developer and don't want to clone the repository, simply run the autoortho executable from the releases-download.

7. How do I know it's working properly?
Well, if you load into the sim and your ground textures look like satellite imagery, it's working ;). You can also check the cache folder which you configured and see if there are any JPEG files which have been downloaded.

**Reply by sergiolamares** [reply]
Thanks for the very detailed explanation to my quetions.

> 3. On the CMD window i'm always getting a message saying "INFO:flighttrack:We are connected but a packet timed out. NBD." what does it mean?
>    It basically means that the flight tracker from autoortho could not get the current airplane position from the sim. NBD means "no big deal", you can safely ignore it. The message will escalate if autoortho can't communicate with the sim for a longer time.

What do you mean by "escalate"?
I noticed the tiles do not match the roads overlays, Is it possible it is downloading the wrong ones because it's not getting the current airplane position?

Thanks once again

**Reply by evilstiefel** [reply]
It would say `FT: Flight disconnected.` instead of the NBD-message.

And no, it's highly unlikely there would be wrong tiles. Either you have the correct tiles for your current location, or no tiles for that location. This sounds more like something related to your autogen and order of things in the `scenery_packs.ini`, unrelated to what autoortho does.

**Reply by sergiolamares** [reply]
> It would say `FT: Flight disconnected.` instead of the NBD-message.
> 
> And no, it's highly unlikely there would be wrong tiles. Either you have the correct tiles for your current location, or no tiles for that location. This sounds more like something related to your autogen and order of things in the `scenery_packs.ini`, unrelated to what autoortho does.

Please take a look at this thread in x-plane.org forum https://forums.x-plane.org/index.php?/forums/topic/285753-autoortho/ 
Looks like all z_AAA_## folders have mixed tiles from different continents, according to data in those folders... is this the proper way autoortho works? it seems unlogical to me, but i'm not the expert here.
Thanks.

**Reply by sergiolamares** [reply]
@kubilus1 ? Anyone ?

**Comment by tbarry0210** [comment]
Question>>>>>

Do I occassionally delete my cache? If I have the storage is it better to keep the files on my SSD or delete and let the files stream? Thank you

**Reply by evilstiefel** [reply]
I don't and you shouldn't have to if space is not a concern. It just keeps autoortho from redownloading the (potentionally) same files if they weren't changed by the provider. So you save time and bandwidth.
